tests/jest/components/AdminSettings.spec.js

```diff
@@ -82,7 +91,7 @@ describe('AdminSettings.vue', () => {
 	})
 	const confirmSpy = jest.spyOn(global.OC.dialogs, 'confirmDestructive')
 
-	describe('form mode and completed status', () => {
+	describe('form mode and completed status without group folder, user app password setup', () => {
```

Comment:
I think the app password comes with the group folder setup so maybe not needed to be mentioned. What do you think?
```suggestion
	describe('form mode and completed status without group folder setup', () => {
```

Reply:
yes can be removed but still can be checked different the state of it.